__tests__/app.test.js

```diff
@@ -185,3 +187,102 @@ describe('GET /api/articles/:article_id/comments', () => {
       });
   });
 });
+
+describe('POST /api/articles/:article_id/comments', () => {
+  it('200: creates a new comment for a valid article id and responds with the new comment object', () => {
```

Comment:
This should be a status 201 - for a created comment object 😉

Reply:
Good catch!